cmd/summary.go

```diff
@@ -120,7 +120,7 @@ func GetSummaryCommand() *cobra.Command {
 
 							doneChan <- true
 
-							return
+							return err
```

Comment:
This won't work, line 144 starts a new go routine, which executes this code. What is being returned here is a nil `err` value from inside an async thread. 

returning `err` here will return nothing, the thread will exit and continue executing on line 159. The return is actually on line 163.

There are a number of the error returns that need to be handled, before returning an error if there were any breaking changes, you will need to check for breaking changes in the model and then return an error if there were breaking changes.